### ISSUE TITLE
Hitting enter when text input for chat is empty should close chat again

### ISSUE THREAD

**Issue by tcmg** [tcmg/adventure-issues#56]
**Is your feature request related to a problem? Please describe.**
I expect hitting enter to unfocus chat when text input is empty

**Describe the Idea or Solution you'd like**
Hitting enter when text input for chat is empty should close chat again.

**Why?**
A few people said they expected it, it worked well in prototype too.

**Comment by ChristofferBech** [comment]
I agree. This is also on our to-do list. I will bump the priority up
